components/apimgt/org.wso2.carbon.apimgt.impl/pom.xml

```diff
@@ -378,7 +378,6 @@
                             org.apache.axis2.*; version="${imp.pkg.version.axis2}",
                             org.apache.commons.*,
                             org.apache.http.*,
-                            org.apache.synapse.*,
```

Comment:
why  do we need to remove this?

Reply:
This has done to fix https://wso2.org/jira/browse/APIMANAGER-4038.